### ISSUE TITLE
Winetricks show me installable application even if it not fit into the architecture

### ISSUE THREAD

**Issue by Mannshoch** [Winetricks/winetricks#1045]
**Step to reproduce.**

1. Start winetricks inside an 64bit Wine
1. try install dotnet35
1. error appear "This package (dotnet35) does not work on a 64-bit installation. You must use a prefix made with WINEARCH=win32."

**expected:**

1. dotnet35 is marked as not installable in the application list.
1. Information about workaround, how-to, description why, how to contribute, etc.

**Comment by austin987** [comment]
Yeah, I've thought about this before, haven't gotten around to fixing it yet as it would involve a bit of refactoring. I think I may have done some work on this when looking at arm support, will have to find that branch..

**Comment by Mannshoch** [comment]
Little Question. Windows is able to start 32bit software also in 64bit Windows. Why is there a limitation in installing 32bit .Net inside 64bit wine?

**Comment by austin987** [comment]
Bugs in wine.

**Comment by Mannshoch** [comment]
What a surprise.

It  would be nice if Winetricks is able to perform some checks to see which functionality are already present, like mono vs. dotnet. It's sometimes difficult to get what software you have to install and what is already present.

**Comment by austin987** [comment]
I'm not sure what you mean by that (but it would probably be a different issue, anyway).

**Comment by austin987** [comment]
Slight addendum, it can also be that the installer itself doesn't support 64bit.
